packages/kad-dht/test/rpc/handlers/find-node.spec.ts

```diff
@@ -44,21 +38,33 @@ describe('rpc - handlers - FindNode', () => {
     })
   })
 
-  it('returns self, if asked for self', async () => {
+  it('returns nodes close to self but not self, if asked for self', async () => {
```

Comment:
This is a bit difficult to read
```suggestion
  it('returns nodes close to self but excludes self, if asked for self', async () => {
```